pkg/cli/bootstrap.go

```diff
@@ -0,0 +1,255 @@
+// Copyright 2021 Outreach Corporation. All Rights Reserved.
+
+// Description: This file contains cli functions used in bootstrap
+// and eventually in stencil.
+package cli
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/signal"
+	"os/user"
+	"runtime"
+	"runtime/debug"
+	"strings"
+	"syscall"
+
+	"github.com/getoutreach/gobox/pkg/app"
+	"github.com/getoutreach/gobox/pkg/cfg"
+	"github.com/getoutreach/gobox/pkg/exec"
+	"github.com/getoutreach/gobox/pkg/log"
+	"github.com/getoutreach/gobox/pkg/secrets"
+	"github.com/getoutreach/gobox/pkg/trace"
+	"github.com/getoutreach/gobox/pkg/updater"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+	"gopkg.in/yaml.v2"
+)
+
+const (
```

Comment:
const block needs a comment for lintroller platinum, even tho this isn't platinum we might as well get with the program